libs/loopsidesheet/src/lib/ui-sidesheet/LoopContentTable.tsx

```diff
@@ -66,35 +65,11 @@ const columns: ColDef<LoopContent>[] = [
   {
     headerName: 'Comm Pkg',
     valueGetter: (pkg) => pkg.data?.commissioningPackageNo,
```

Comment:
The loop/id/content endpoint returns both commpkgurl and mcpkg url. I am not sure why these were removed from the table(prior to this pr). Could you take a quick look at adding links?